libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -508,6 +508,104 @@ class Strings {
         assertPrints(mixedColor, "brown&blue")
     }
 
+    @Sample
+    fun toPattern() {
+        val string = "this is a regex"
+        val pattern = string.toPattern(1)
+        assertPrints(pattern.pattern(), string)
+        assertTrue(pattern.flags() == 1)
+    }
```

Comment:
It would be nice to show that the pattern matches everything once constructed. I mean, why otherwise one would instantiate a pattern, right? :)